tests/integration/baseTestValidator.py

```diff
@@ -104,7 +104,6 @@ def run_test(self, jobId, statusId, statusName, fileSize, stagingRows,
             stagingRows: Expected number of rows in validation db staging tables. False if no rows are expected
             errorStatus: Expected status in file table of error DB, False if file object should not exist
             numErrors: Expected number of errors
-            rowErrorsPresent: Checks flag for whether row errors occurred, None to skip the check
```

Comment:
these kinds of docstrings are being removed because they are no longer provided as part of the input to the function call correct?

If we're removing the ones that aren't used, shouldn't we also add the ones that aren't present? For example, `numWarnings` and `warningFileSize`